src/components/AudioPlayer/Audio/Audio.component.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const AudioComponent = ({ autoPlay, songLink }) => (
+  <div className="audio">
+    <audio controls autoPlay={autoPlay}>
+      <track kind="captions" />
+      <source src={songLink} type="audio/mpeg" />
+      <source src={songLink} type="audio/wav" />
+      <source src={songLink} type="audio/ogg" />
+    </audio>
+  </div>
+);
+
+AudioComponent.propTypes = {
+  autoPlay: propTypes.bool,
+  songLink: propTypes.string,
+};
+
+AudioComponent.defaultProps = {
+  autoPlay: true,
+  songLink: propTypes.string.isRequired,
+};
```

Comment:
This is all wrong:

```js
AudioComponent.propTypes = {
  autoPlay: propTypes.bool,
  songLink: propTypes.string.isRequired,
};

AudioComponent.defaultProps = {
  autoPlay: true,
};
```

I'd remove the `defaultProps` and make the `autoPlay` prop required, tho.

Reply:
In line 23 you are declaring `songLink` will default to a function actually. This is wrong.